### ISSUE TITLE
Try nodemon?

### ISSUE THREAD

**Issue by kentcdodds** [sarbbottam/watch-and-exec#5]
How does this differ from npm.im/nodemon?

**Comment by sarbbottam** [comment]
**This module is a by-product of [watch-and-rsync](https://github.com/sarbbottam/watch-and-rsync)**.

TL;DR
---

I was aware of `nodemon` but [did not know it support non hanging process too](https://www.npmjs.com/package/nodemon#automatic-re-running)

I have been seeking answer for  [nodejs - watch directory for changes in background and perform some action](http://stackoverflow.com/questions/36584637/nodejs-watch-directory-for-changes-in-background-and-perform-some-action), I wish I would have got a reminder for [nodemon](npm.im/nodemon) then.

---

I tried to use `nodemon` as suggested, however out of luck, 

```sh
nodemon --exec="rsync -azOte  --delete --exclude node_modules ~/GitHub/watch-and-rsync/test/fixtures/source/ ~/GitHub/watch-and-rsync/test/fixtures/target" --watch /Users/sabandyo/GitHub/watch-and-rsync/test/fixtures/source 
[nodemon] 1.9.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/sabandyo/GitHub/watch-and-rsync/test/fixtures/source/**/*
[nodemon] starting `node "--exec=rsync -azOte  --delete --exclude node_modules ~/GitHub/watch-and-rsync/test/fixtures/source/ ~/GitHub/watch-and-rsync/test/fixtures/target" src/index.js`
node: bad option: --exec=rsync -azOte  --delete --exclude node_modules ~/GitHub/watch-and-rsync/test/fixtures/source/ ~/GitHub/watch-and-rsync/test/fixtures/target
```
---

I guess I have to invest some time to figure out how `nodemon` can address my use case.
Since the existing solution suffice my need I'm sticking to it. Happy to abandon this in favor of `nodemon`, if I can get [watch-and-rsync](https://github.com/sarbbottam/watch-and-rsync) work with `nodemon`
